pkg/node/sendoperation/sendoperation.go

```diff
@@ -2,36 +2,39 @@ package sendoperation
 
 import (
 	"context"
-	"crypto/rand"
+	"crypto/ed25519"
 	"encoding/binary"
 	"fmt"
+	"strings"
 
-	"github.com/btcsuite/btcd/btcec/v2"
```

Comment:
You should perform a `go mod tidy` to remove unused libraries from go.mod and go.sum files.